### PR TITLE
Fix ordering of Scheme toplevel attrs

### PR DESCRIPTION
**Pull request by maxwellE**
Fix for incorrect ordering of `Scheme` attrs when
using `wasCreatedForAppExtension` value

### CODE REVIEW

File: Tests/XcodeProjTests/Extensions/AEXML+XcodeFormatTests.swift [hunk 1]
```diff
@@ -121,6 +131,30 @@ class AEXML_XcodeFormatTests: XCTestCase {
...
+            childName: "Scheme",
+            attributes: [
+                "LastUpgradeVersion": "1320",
+                "wasCreatedForAppExtension": "YES",
+                "version": "1.7"
+            ]
+        )
+    }
+
+    func test_Scheme_attributes_sorted_when_original_unsorted() {
```
Comment (kwridan) [inline Tests/XcodeProjTests/Extensions/AEXML+XcodeFormatTests.swift:146]: minor: this test may be redundant, the order in which dictionary elements are specified to my knowledge is opaque to the underlying types, the dictionaries are equal between this test and `test_Scheme_attributes_sorted_when_original_sorted`, as such only one of those tests will suffice.

An interesting test could be the following (but may be overkill for this)

- Start with an unsorted raw XML (string) scheme
- Deserialize it (read it vis `XCScheme`)
- Serialize it back to XML
- Verify the correct sort order

**Review by kwridan — APPROVED** [review]
Thanks @maxwellE

**Review by luispadron — APPROVED** [review]

**Comment by luispadron** [comment]
@maxwellE thanks for the PR. I've merged some of them and looks like that created a conflict here. Once that's resolved we can get this merged!

**Comment by luispadron** [comment]
@all

**Pull request merged**